l10n_ch_account_statement_base_import/parsers/base_parser.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#    Copyright 2015 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from abc import ABCMeta, abstractmethod
+import logging
+from openerp import _, exceptions
+_logger = logging.getLogger(__name__)
+
+
+class BaseSwissParser(object):
```

Comment:
I am not sure, @nbessi did the port from version 8 where the parsers were pure python objects. I don't know how AbstractModels work, won't it create a table in the database for each parser then ?